src/js/view/components/LogViewerList.js

```diff
@@ -130,12 +130,19 @@ const LogViewerList = props => {
     );
   };
 
+  //listRef is used to get the reference to the List object so that we can use its method forceUpdate
+  const listRef = useRef();
```

Comment:
Nice fix. The wrapping happens directly now. Just one thing I would change is to move this ref constant to the top of the component function, among the other ref hooks.

Reply:
I agree with this!